README.md

```diff
@@ -119,9 +119,9 @@ preflight check --config-file=./examples/pods.preflight.yaml
 ```
 
 You will see a CLI formatted report if everything goes well.
-Also, you will get a JSON report in `./output`. 
+Also, you will get a JSON report in `./output`.
 
-## Use Preflight Web UI
+### Use Preflight Web UI
```

Comment:
Why is this not rendering properly?

Reply:
Seems to be working now... Not sure why. I replaced the `#`s which implies there are different types of `#`.